internal/proxy/providers/sso_test.go

```diff
@@ -201,11 +210,12 @@ func TestSSOProviderGroups(t *testing.T) {
 
 func TestSSOProviderGetEmailAddress(t *testing.T) {
```

Comment:
nit, unrelated to the changes but this function name should probably be TestRedeem since it doesn't call `GetEmailAddress` and calls the Redeem function.

Reply:
changing it